pkgs/top-level/python-packages.nix

```diff
@@ -9140,6 +9140,27 @@ in {
     };
   };
 
+  td_watson = buildPythonApplication rec {
```

Comment:
`python-packages.nix` is for Python libraries and not for applications that happen to be written in Python. Please move it elsewhere in the Nixpkgs tree.